migrations/etl/1_Mine_profile_detail_location.sql

```diff
@@ -147,6 +150,45 @@ BEGIN
         (new.lat_dec <> 0 AND new.lon_dec <> 0);
     SELECT count(*) FROM mine into new_row;
     SELECT count(*) FROM mine_location into location_row;
+
+    -- Upsert data from new_record into mine_type
+    WITH new_record AS (
+        SELECT
+            mine_guid,
+            mine_typ
+        FROM ETL_PROFILE
```

Comment:
It looks like ETL_PROFILE is only populated when a mine in MMS has not already been created in MDS. I think this will be fine for the upcoming release as we are planning on wiping and repopulating production data, but this pattern will not work in the future if we need to pull in new data for mines that already exist.

Reply:
Yeah, there was a discussion this morning about the plan for migrating to MDS as the source of truth. Apparently this discussion has been happening since March. When a business decision has been made on it, we can update ETL to exclude certain fields / pull all mines / etc.

Reply:
Personally, I find that unlikely as there will be a lot of functionality not implemented in MDS and users will likely continue to use and update data in MMS for years. That discussion is way outside of the scope of this ticket :) Thanks!